priam/src/main/java/com/netflix/priam/utils/TokenManager.java

```diff
@@ -40,9 +42,9 @@ public String createToken(int mySlot, int availabilityZones, int availabilityZon
     public abstract String sanitizeToken(String jmxTokenString);
 
     /**
-     * Create an offset to add to token values by hashing the region name.
+     * Create an offset to add to token values by hashing the location.
      */
-    public static int regionOffset(String region) {
-        return Math.abs(region.hashCode());
+    public static int locationOffset(Location location) {
+        return Math.abs(location.hashCode());
```

Comment:
minor: Related to the tests: there are tests named regionOffset() in 3 Test files. should we clear them or replace with LocationOffset type tests?